### ISSUE TITLE
STX stuck after failed transaction

### ISSUE THREAD

**Issue by bitcoincidence2** [leather-io/extension#1506]
After a failed transaction the amount of STX seems to be stuck. I cant send it to another wallet though it is shown correct in the balance.
Stacks Wallet v2.12.4
adress: SPFPH52MQP00S4880WFBP72GCHMKBQVRE5QYSTFG
current balance: 138 STX
max amount sendable: 18 STX

**Comment by aulneau** [comment]
related: https://github.com/blockstack/stacks-wallet-web/issues/1488

You have a locked balance (stacked)

![Screen Shot 2021-07-29 at 9 51 59 AM](https://user-images.githubusercontent.com/11803153/127514334-60a915fb-98df-4c81-9453-25345afe7f91.png)

**Comment by ikill4food2** [comment]
Believe I am having same issue using hiro web wallet and have a permanently stuck wallet address - it will not send or receive tokens and any transaction sits in mempool until dropoff at 256 blocks. 
There is nothing seen in the pool that keeps failing this address.
The last error occured while using btc.us site to change website URL redirect  - transaction failed and address has been stuck since. 
Also, nothing happens/no response when attempts using btc.us to transfer or change, . Same issue.

When this latest test transaction of 1 STX drops from pool, the address still will not receive or send STX. 


SP3KZ6BFDWW3NY2RS0WATQ3J360HA41WJFY0DVJWG

Thank you

![error1](https://user-images.githubusercontent.com/25948909/127773071-06ac0cba-6757-4dc8-b2aa-506836f821c6.jpg)

**Comment by markmhendrickson** [comment]
@ikill4food2 your issue here appears different in that there doesn't appear to be a discrepancy between the balance shown and how much you can send, is there? Simply any transaction you try fails?

**Comment by ikill4food2** [comment]
Hello, Yes ... everything fails.

**Comment by markmhendrickson** [comment]
This is related to https://github.com/blockstack/stacks-wallet-web/issues/1488. We'll need to investigate further.

**Comment by bitcoincidence2** [comment]
anything new on my case? Can i do something?

**Comment by ikill4food2** [comment]
Important update - after the 1 STX transaction fell off the pool, my balance is now showing incorrectly in max send vs. available.
Looks to be almost same issue now? I will wait on trying any further transactions as they will just fail.
Hope this in some way helps.

![error4](https://user-images.githubusercontent.com/25948909/127979293-2cbbf239-291c-4f2a-ba27-170b320c0ae7.jpg)

**Comment by markmhendrickson** [comment]
@bitcoincidence2 There aren't any updates yet. Per https://github.com/blockstack/stacks-wallet-web/issues/1506#issuecomment-891104048, we'll need to investigate further first.

@ikill4food2 to be clear here, the value "10000" is entered into the field automatically upon clicking "Max"? Or you typed "10000" yourself manually?

https://explorer.stacks.co/address/SP3KZ6BFDWW3NY2RS0WATQ3J360HA41WJFY0DVJWG?chain=mainnet

**Comment by ikill4food2** [comment]
@markmhx I was typing the 10000 manually --- but, I now see it will display if I just simply press the max button  ... Here is a clean screen shot of using the max button without entering anything in the fields.

![error5](https://user-images.githubusercontent.com/25948909/128044272-420751f0-1a9f-4a3a-81d6-9dd889d1d837.jpg)

**Comment by markmhendrickson** [comment]
@ikill4food2 I'm not sure I see the problem here. The amount entered (6.997331) appears to be your balance minus the fees?

**Comment by ikill4food2** [comment]
@markmhx There is nothing being sent, and nothing pending in the pool ... yet the balances are different...
the very main problem right now is...
If I do a transaction, any transaction to send or receive - It will fail.. again and again.
 I cannot send or receive anything to this address as it is stuck.
I can do another transaction and show you...again ... if you like.

**Comment by aulneau** [comment]
@ikill4food2 I'm confused -- you need to enter an address to be able to send anything. Am I missing something?

**Comment by ikill4food2** [comment]
@aulneau The address is stuck ... I cannot send or receive anything to or from this address after the last error I mentioned in above comments.

**Comment by aulneau** [comment]
> The address is stuck ... I cannot send or receive anything to or from this address after the last error I mentioned in above comments.

This is technically impossible :) anyone can send to any address. there is no concept of "stuck"

**Comment by ikill4food2** [comment]
@aulneau You see the screenshots above ??? I cannot send or receive to the address anymore... the STX is stuck in the wallet.
If I try to send STX to the address, the trasnsaction sits as pending until it drops off. 
Same for sending.
I have tried to send STX to this wallet and it is Stuck ... that is the only way I know how to put it. 
ANy suggestions ???

**Comment by ikill4food2** [comment]
@aulneau If I send another 1 STX to this address from any other wallet ... it fails.
Nor can I transfer any out ... It also sits in pending until it drops from the pool.

**Comment by ikill4food2** [comment]
Please, be my guest, anyone try to send any amount of STX to the address ... it will just sit in pending like the many other times I have tried before reporting this. 
Nor can I send anything out from this address.
I was trying to show this and explain in screenshots above ... sorry for any confusion
I seriously cannot access or do anything with this address anymore.

SP3KZ6BFDWW3NY2RS0WATQ3J360HA41WJFY0DVJWG

**Comment by bitcoincidence2** [comment]
@ ikill4food2
**But maybe you should open your own issue?**
I have the feeling my case is not in the foreground anymore  ;-) 

But some more facts to may case:
I tried to send STX to a Pool to which i had already send. So the transaction failed with (err 3)
I did this 4 times and i went over this page
https://stacks-pool-registry.pages.dev/pools

here i chose the following:

friedgerpool.id verified

SP700C57YJFD5RGHK0GN46478WBAM2KG3A4MN2QJ
Locking for 12 cycles.
Minimum amount required to join: 40 STX
Payout in STX.
When payout? 2 days after last reward received.
Fees: none
Reward addresses:
33WSGLeVoEpuZDjB54HKZ1y5YsERELoVNq
1AJm7XWhbPTFqgtJT6oS9wQ3XThk7rA2yF
Using contract:
SP000000000000000000002Q6VF78.pox

**Comment by ikill4food2** [comment]
@bitcoincidence2 They will advise if this needs to be moved to separate issue.
 As of now, we both have "STX stuck after failed transaction" and cannot send...could possibly still be related.

**Comment by ikill4food2** [comment]
Happy to report that I was able to receive & send STX again...just passed both tests - great support , I appreciate all of the quick replies & suggestions. 
Thank you.
@bitcoincidence2 last thing i tried was receiving another 1 STX to my address, after confirm - was then able to send STX. 
hopefully yours will work now as well.

**Comment by bitcoincidence2** [comment]
@ikill4food2: thanky, i will try - but i guess my case is somehow different...

**Comment by fbwoolf** [comment]
@markmhx I have a PR up for #1488 should I add this as a linked issue?
https://github.com/blockstack/stacks-wallet-web/pull/1521

**Comment by bitcoincidence2** [comment]
If this is a wallet problem can i try another wallet?

Now also strange:
Bevore the incident i sent 120 stx successful to this pool. They are not shown in the balance of the wallet (on page 1) anymore (i guess because they are in the pool).
But if i change to the "send" view (by clicking on send) it shows a balance that contains this 120 stx that are in the pool, too. This balance on "send" is all i ever had in the wallet. Which makes also no sense - at least on the "send" page - because even if pooled stx are somehow a part of the wallet i surely cant send them.

Could my problem be that the 120 stx  i can't send (but are shown in the balance) did not come back to me though there was this (err 3) error? But why are in the balance?

Sorry for the confusion that i have two 120 stx transactions. One successful to the pool and one that seems to be stuck. (next time i will make the second one at least a 121...)

**Comment by markmhx** [comment]
@ikill4food2 please do open a new issue and include any and all links to your transactions on [the Explorer](https://explorer.stacks.co/?chain=mainnet) that are pending or failed for context: https://github.com/blockstack/stacks-wallet-web/issues/new

@bitcoincidence2 thanks for the extra details, we'll follow up once we've had a chance to investigate 🙏

**Comment by markmhendrickson** [comment]
@fbwoolf please do link that PR if it also resolves this particular issue 🙏

**Comment by bitcoincidence2** [comment]
Is there a new version of the wallet i could try?

**Comment by bitcoincidence2** [comment]
I installed this new version 
 "fix: inconsistent balances"
and now it works!
thx for your help!

**Comment by fbwoolf** [comment]
> I installed this new version
> "fix: inconsistent balances"
> and now it works!
> thx for your help!

Fantastic, good to know! Hoping to get that PR merged asap.

**Comment by Eshwari007** [comment]
Great,I was not able to reproduce the issue as well.Closing the ticket
